src/components/molecules/button/button.ts

```diff
@@ -0,0 +1,103 @@
+import { LitElement, html } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+import type { MsIcons } from '~/assets/icons'
+import { Icons } from '~/assets/icons'
+import { WithTheme } from '~/mixins'
+import { getThemeColor } from '~/utils'
+
+import { msButtonstyles } from './button.styles'
+
+@customElement('ms-button')
+export class MSButton extends WithTheme(LitElement) {
+  static styles = msButtonstyles
+
+  @property({ type: String })
+  variant = 'contained'
+
+  @property({ type: String })
+  color =
+    this.variant === 'contained'
+      ? getThemeColor('gray-white').toString()
+      : getThemeColor('regular-user-primary-main').toString()
+
+  @property({ type: Boolean })
+  disabled = false
+
+  @property({ type: String })
+  ariaLabel = ''
+
+  @property({ type: String })
+  size = 'small'
+
+  /**
+   * Set position for any icon, custom or preloaded one
+   * @type {string}
+   * @default 'left'
```

Comment:
2 things:
1 - why some @properties has jsdoc and others no?
2 - does this really work if you use it in a property? In my opinion is better to document in class eg `ms-tag`

Reply:
done!